src/hooks/usePantry.ts

```diff
@@ -181,12 +181,50 @@ function entry({ project }: { project: string }): Entry {
     const versions = dir.join("versions.txt")
     return { dir, yml, versions }
   }
-  throw new TeaError('not-found: pantry: package.yml', {project})
+  
+  const closestPkg = await getClosestPackageSuggestion(project);
```

Comment:
let’s not do this here, instead do it in the error handler in `app.ts`. This way we don’t have to change everything from sync to async. diff will be smaller, and the error handler is really the right place to do this. Once this code is librified it really should be up to the caller whether or not they want to do this.

Reply:
I guess actually it'll be a little tricky with the current infra, so I'll do it after the merge.

Let’s get tests passing and great! Thanks so much for this contribution.